site/docs/platforms.md

```diff
@@ -113,3 +113,117 @@ command-line flags:
 
 *  `--host_platform` - defaults to `@bazel_tools//platforms:host_platform`
 *  `--platforms` - defaults to `@bazel_tools//platforms:target_platform`
+
+## Skipping incompatible targets
+
+When building for a specific target platform it is often desirable to skip
+targets that will never work on that platform. For example, your Windows device
+driver is likely going to generate lots of compiler errors when building on a
+Linux machine with `//...`. Use the
+[`target_compatible_with`](be/common-definitions.html#common.target_compatible_with)
+attribute to tell bazel what target platform constraints your code has.
```

Comment:
Should "bazel" be capitalized (here and elsewhere in the docs)?

Reply:
Done